src/Files.App/Views/Settings/ActionsPage.xaml

```diff
@@ -410,5 +411,13 @@
 			PreferredPlacement="Auto"
 			Subtitle="{helpers:ResourceString Name=KeybindingAlreadyUsedNotification}" />
 
+		<!--  Invalid Key TeachingTip  -->
+		<TeachingTip
```

Comment:
Can you send a screenshot? Also, is there a way I can display this?

Reply:
https://github.com/files-community/Files/pull/15420#issuecomment-2118429334

Reply:
If pressed down one of this or if not localised aka it's string.Empty
```csharp
// Define invalid keys that shouldn't be processed
var invalidKeys = new HashSet<VirtualKey>
{
	VirtualKey.CapitalLock,
	VirtualKey.NumberKeyLock,
	VirtualKey.Scroll,
};
```

https://github.com/files-community/Files/assets/84145589/5b6f9aba-7829-4ce7-8673-29b1045eb05b

Reply:
Thanks